eventually/eventually/urls.py

```diff
@@ -13,9 +13,8 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
-from django.contrib import admin
+from django.conf.urls import url, include
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    url(r'^api/v1/team/', include('team.urls')),
```

Comment:
have we needed changes in this file? have we changed them?

Reply:
done